medusa/search/manual.py

```diff
@@ -276,7 +276,8 @@ def get_provider_cache_results(indexer, show_all_results=None, perform_search=No
         cached_results = [dict(row) for row in sql_total]
         for i in cached_results:
             i['quality_name'] = Quality.split_quality(int(i['quality']))
-            i['time'] = datetime.fromtimestamp(i['time']).strftime(app.DATE_PRESET + ' ' + app.TIME_PRESET)
+            i['time'] = datetime.fromtimestamp(i['time'])
+            i['time_pretty'] = pretty_date(i['time'])
```

Comment:
@fernandog I know you were also keeping the original date.  My point was that creating a variable for the pretty date is unnecessary, the function should just be applied to the UI.  Also I think it should be treated like the option for series times to be in local time or network timezone.